src/modules/Dashboard/components/CardImage/CardImage.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+
+import { toggleNotesModal, updateModalValue, deleteCurrentImage } from "modules/Dashboard/actions/dashboard";
+import { Button, Card } from "react-bootstrap";
+
+import {
+    faQuestionCircle,
+    faExclamationCircle,
+    faHourglass,
+    faCheck,
+    faHammer,
+    faStickyNote,
+    faTrash,
+    faDownload,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const CardImage = ({
+    handleDeleteImage,
+    handleOpenModal,
+    imageId,
+    userImage,
+}) => {
+    const startedAtParse = (startedAt) => {
+        const date = new Date();
+        date.setTime(Date.parse(startedAt));
+        return date.toGMTString();
+    };
+
+    const statusIcon = {
+        Undefined: faQuestionCircle,
+        Pending: faHourglass,
+        Building: faHammer,
+        Failed: faExclamationCircle,
+        Succeeded: faCheck,
+    };
+
+    return (
+        <Card style={{ height: "calc(100% - 1rem)" }}>
+            <Card.Body>
+                <div className="row">
+                    <div className="col-xs-12 col-sm-4 col-md-6">
+                        <h4 className="text-dark">
+                            {`${userImage.distro_name}
+                            for
+                            ${userImage.device_name}
+                            `}
+                        </h4>
+                        <p className="mb-1">{`Build type: ${userImage.flavour_display}`}</p>
```

Comment:
May be worth wrap blocks in `div` and set him only one `mb-1`?

Reply:
Bootstrap make too big margin-bottom for `<p>`, wrap blocks in div don't make mb smaller